tap_anthem/tap.py

```diff
@@ -4,51 +4,24 @@
 
 from singer_sdk import Tap, Stream
 from singer_sdk import typing as th  # JSON schema typing helpers
-# TODO: Import your custom stream types here:
 from tap_anthem.streams import (
-    anthemStream,
-    UsersStream,
-    GroupsStream,
+    NhPlansStream,
+    NhProvidersStream,
+    NhDrugsStream
 )
-# TODO: Compile a list of custom stream types here
-#       OR rewrite discover_streams() below with your custom logic.
+
 STREAM_TYPES = [
-    UsersStream,
-    GroupsStream,
+    NhPlansStream,
+    NhProvidersStream,
+    NhDrugsStream
 ]
 
 
 class Tapanthem(Tap):
     """anthem tap class."""
     name = "tap-anthem"
 
-    # TODO: Update this section with the actual config values you expect:
-    config_jsonschema = th.PropertiesList(
-        th.Property(
-            "auth_token",
-            th.StringType,
-            required=True,
-            secret=True,  # Flag config as protected.
-            description="The token to authenticate against the API service"
-        ),
-        th.Property(
-            "project_ids",
-            th.ArrayType(th.StringType),
-            required=True,
-            description="Project IDs to replicate"
-        ),
-        th.Property(
-            "start_date",
-            th.DateTimeType,
-            description="The earliest record date to sync"
-        ),
-        th.Property(
-            "api_url",
-            th.StringType,
-            default="https://api.mysample.com",
-            description="The url for the API service"
-        ),
-    ).to_dict()
+    config_jsonschema = th.PropertiesList().to_dict()
```

Comment:
If this tap were made more generic in the future (i.e., to support multiple states beyond NH), this would be a good place to pass in a list of specific states to pull data for. 

Since that is not a use case today, and the Anthem dataset is available publicly, I removed this boilerplate config.